src/components/Tap.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+    <main class="container" id="tap">
+
+        <div class="content">
+
+            <h1>Add a voucher</h1>
+
+            <form id="textVoucher" v-on:submit.prevent>
+                <label for="voucherBox" id="lblVoucherBox">Type a voucher code</label>
+
+                <div class="input-box">
+                  <input id="sponsorBox"
+                         @keypress='onChangeSponsorBox'
+                         type="text"
+                         v-model="sponsorCode"
+                         ref="sponsorBox"
+                         maxlength="3"
+                  >
+                  <input id="voucherBox"
+                         v-on:keyup.delete='onDelVoucherBox'
+                         type="tel"
+                         pattern="[0-9]*"
+                         v-model="voucherCode"
+                         ref="voucherBox"
+                         maxlength="8"
+                  >
+                </div>
+
+                <button v-on:click="record" id="submitVoucher">Add</button>
+
+            </form>
+
+            <div id="registeredVouchers" v-if="recVouchers.length > 0">
+                <h2>Your recorded vouchers</h2>
+                <ul id="recVouchersList">
+                    <li v-for="recVoucher in recVouchers[0]">
+                        {{ recVoucher }}
+                    </li>
+                </ul>
+            </div>
+
+        </div>
+
+    </main>
+</template>
+
+<script>
+/* Copyright (c) 2017, Alexander Rose Charity (reg. in England and Wales, #00279157) */
```

Comment:
I took it out because it's not needed on that screen, or anywhere as far as I'm aware - but if we do need it further down the line I can put it back in.